backend/middleware/jwtMiddleware.js

```diff
@@ -0,0 +1,27 @@
+const jwt = require('jsonwebtoken');
+const jwtSecret = process.env.JWT_SECRET;
```

Comment:
May we please move this line to be after the module imports, since it's not really a module. i think placing it after line 5 is better